### DISCUSSION TITLE
I created a new account, Do I need to delete the old one?

### DISCUSSION THREAD

**Discussion by nelliesnoodles** [community/community#22910] [New to GitHub]
<p>I was flagged today on my account under my currently used e-mail that I started a new github account under,  I’m thinking it’s because I never removed the old account.   Do I need to go in and delete the old one (under a different e-mail server)?  I have no problem deleting the old one if that is what I need to do.  </p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/nelliesnoodles">@nelliesnoodles</a>,</p>
<p>For help with account-specific and private information, you’ll need to reach out to private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>. As a general rule though, it’s actually against the GitHub Terms of Service to have more than one free account. I’d recommend reaching out to support for further help!</p>

**Comment by nelliesnoodles** [comment]
<p>Thank you.  I have deleted the old account.  I was in the process of migrating everything away from that old e-mail address.   Thanks so much. </p>

**Comment by OLUBEST** [comment]
<p>GOOD</p><p> </p>
